java/src/jmri/server/json/JsonClientHandler.java

```diff
@@ -83,7 +83,7 @@ public void onClose() {
      * @see #onMessage(JsonNode)
      */
     public void onMessage(String string) throws IOException {
-        if (string.equals("{\"type\":\"ping\"}")) {
+        if (string.startsWith("{\"type\":\"ping\"")) {
```

Comment:
The equals was no longer getting a match, (since "method" is now part of the string), so debugging was more difficult because of the recurring "ping"s.

Reply:
The code *was* correct; a method parameter is not supposed to be included with a ping message. Please fix whichever client is not working correctly instead of changing this code.

Reply:
@rhwood , This is the change that explicitly adds method to send:
https://github.com/JMRI/JMRI/commit/84cfeb9dd07387e48886a1815a7143552f35c2a5#diff-a1315e7f4b655296f57a66e3968ea948

Reply:
@rhwood This is where the ping is setup for sending, used by various clients:
https://github.com/JMRI/JMRI/blob/master/web/js/jquery.jmriConnect.js#L232
I see no issue with this approach. Please advise.

Reply:
@rhwood Can you post a link to this documented protocol?